src/main/java/com/epam/reportportal/extension/jira/CloudJiraExtension.java

```diff
@@ -64,7 +64,7 @@
 public class CloudJiraExtension implements ReportPortalExtensionPoint, DisposableBean {
 
 	private static final String DOCUMENTATION_LINK_FIELD = "documentationLink";
-	private static final String DOCUMENTATION_LINK = "https://reportportal.io/docs/plugins/JiraCloud";
+	private static final String DOCUMENTATION_LINK = "https://reportportal.io/docs/plugins/AtlassianJiraServer";
```

Comment:
I guess the link should be `https://reportportal.io/docs/plugins/AtlassianJiraCloud`.